test/test_mckay_decomposer.py

```diff
@@ -0,0 +1,93 @@
+import unittest
+
+from opensquirrel import circuit_matrix_calculator, mckay_decomposer
+from opensquirrel.default_gates import *
+from opensquirrel.squirrel_ir import Float, Qubit, SquirrelIR
+
+
+def are_matrices_equal_up_to_global_phase(matrix_a, matrix_b):
```

Comment:
```suggestion
def check_equivalence_up_to_global_phase(matrix_a, matrix_b):
```

Reply:
For me if I rename it that way, the function should return nothing and throw when matrices are not equal up to global phase
So
`check` -> assertion (function throws for instance)
`is` or `are` -> return a boolean, the `False` return value can also be useful in some context


I've renamed it as you suggest but then made it do the `self.assertTrue` on top and return void.